### ISSUE TITLE
Support termux , DNS resolv

### ISSUE THREAD

**Issue by nanqic** [hatoo/oha#463]
In [termux](https://github.com/termux/termux-app)  the DNS file in $PREFIX/etc/resolv.conf .

So got error when run oha in termux :
```
Error: DNS: failed to load /etc/resolv.conf

Caused by:
    io error: No such file or directory (os error 2)
```

May you fit it in termux ? Thank you.

**Comment by hatoo** [comment]
Hi! This is really great issue!

I've implemented something on #464 

Could you try on your side?
If you don't want to build `oha` yourself, you can download a binary from https://github.com/hatoo/oha-release-test/releases/tag/v1.4.4-termux-2

**Comment by nanqic** [comment]
Wow, it works! Thanks a lot for this awesome project!
